src/Miso/Runner.hs

```diff
@@ -4,24 +4,25 @@ module Miso.Runner (run) where
 
 #if defined(wasm32_HOST_ARCH)
 import qualified Language.Javascript.JSaddle.Wasm as J
+#elif defined (ghcjs_HOST_OS)
 #else
 import qualified Language.Javascript.JSaddle.Warp as J
+import           Data.Maybe
 import           System.Environment
+import           Text.Read
 #endif
 
 import           Language.Javascript.JSaddle
 
 -- | Entry point for a miso application
-#if defined(wasm32_HOST_ARCH)
 run :: JSM () -> IO ()
+#if defined(wasm32_HOST_ARCH)
 run = J.run
+#elif defined(ghcjs_HOST_OS)
```

Comment:
Same as above. The `run` introduced by `jsaddle-warp` is identical here, so we can remove this `#elif` case as well and just use a hardcoded port that gets ignored on js backends.

```haskell

-- | Run the given 'JSM' action as the main entry point.  Either directly
--   in GHCJS or as a Warp server on the given port on GHC.
#ifdef ghcjs_HOST_OS
run :: Int -> IO () -> IO ()
run _port = id
#else
run :: Int -> JSM () -> IO ()
run port f =
    runSettings (setPort port (setTimeout 3600 defaultSettings)) =<<
        jsaddleOr defaultConnectionOptions (f >> syncPoint) jsaddleApp
#endif
```